code/game/jobs/job/medical.dm

```diff
@@ -48,7 +48,7 @@
 	idtype = /obj/item/weapon/card/id/medical
 	access = list(access_medical, access_morgue, access_surgery, access_chemistry, access_virology, access_genetics, access_mineral_storeroom)
 	minimal_access = list(access_medical, access_morgue, access_surgery, access_maint_tunnels)
-	alt_titles = list("Surgeon","Nurse","Coroner")
+	alt_titles = list("Surgeon","Nurse","Coroner","General Practitioner")

```

Comment:
I reallyyyy don't think we need another title for medical when it's already bloated to hell and back for alt titles.